### DISCUSSION TITLE
Compression

### DISCUSSION THREAD

**Discussion by barvirm** [tokio-rs/axum#1450] [Q&A] [answered]
Hello, are there any alternative to `tower-http` compression service?
I found out that it is slowing down my server significantly.

Longest request with compression:
<img width="1212" alt="Screenshot_1" src="https://user-images.githubusercontent.com/15948420/194316310-7ef5a0ac-1316-49bb-bcca-69037a218787.png">

Longest request without compression:
<img width="1262" alt="Screenshot_2" src="https://user-images.githubusercontent.com/15948420/194316316-af9afeed-ae3c-49bc-b89a-061656e2a34d.png">

**Comment by davidpdrsn** [comment, marked as answer]
Not that I know of
